src/collective/taxonomy/javascripts/src/js/reducers.js

```diff
@@ -41,6 +43,37 @@ function removeNode(nodes, action) {
   return newNodes
 }
 
+function moveDown(nodes, action) {
+  const swapped = nodes[action.parentId].subnodes.slice(
```

Comment:
Cool ! It would be nice too to disable or hide the move up/down button when it is the first/last item